src/ott/problems/quadratic/quadratic_problem.py

```diff
@@ -285,6 +306,8 @@ def update_linearization(
       epsilon: An epsilon scheduler or a float passed on to the linearization.
       old_transport_mass: Sum of the elements of the transport matrix at the
         previous iteration.
+      remove_scale: Whether to remove any scaling from the cost matrices before
+        computing the linearization.
```

Comment:
here I would specify the only case where this makes sense is for the last iteration of the outer loop. e.g.:

remove_scale: Whether to remove any scaling from the cost matrices when
        computing the linearization of the quadratic cost. At the moment, this is
        only used when doing this update at the last outer iteration of a GW solver.

Reply:
Done!